### PR TITLE
Add License and chat badges

### PR DESCRIPTION
**Pull request by josephca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Codewind ODO extension
+[![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
+[![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
+
 Extension to Codewind providing support for ODO projects: https://codewind.dev
```
Comment (GeekArthur) [inline README.md:6]: For license badge:
Codewind repo uses this one:
[License](https://www.eclipse.org/legal/epl-2.0/)
Codewind appsody repo uses this one:
[License](https://www.eclipse.org/legal/epl-2.0/)
Not sure which one is the right one, but I think we should make it consistent for all Codewind subprojects/repos

If we decide to use this one [License](https://www.eclipse.org/legal/epl-2.0/), then this PR looks good to me

**Review by GeekArthur — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (josephca) [reply README.md:6]: Well, appsody repo should be updated with the same one and a PR was opened already. :) https://github.com/eclipse/codewind-appsody-extension/pull/56

Comment (GeekArthur) [reply README.md:6]: @josephca Thanks, approved this PR.

Comment (josephca) [reply README.md:6]: BTW, please add a build badge when you're ready to add Jenkinsfile. Thanks,

**Review by GeekArthur — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (GeekArthur) [reply README.md:6]: @josephca Yes, will do that with the Jenkins build PR.

**Pull request merged**